bin/simbagen.py

```diff
@@ -168,11 +168,13 @@ def __init__(self, filename, endianess):
         self.endianess = endianess
         self.settings = OrderedDict()
 
-        if filename is None:
+        if not filename:
             return
 
         settings_parser = ConfigParser()
-        settings_parser.read(filename)
+        if not settings_parser.read(filename):
+            print("Failed to load config file from {}".format(filename))
+            return
```

Comment:
Nice catch, but should we just call `sys.exit("Failed to load config file '{}'.".format(filename))` instead of the print and return?

If the settings file parsing failed this script should fail.